src/wp-includes/option.php

```diff
@@ -596,7 +596,7 @@ function wp_load_alloptions( $force_cache = false ) {
 
 	if ( ! $alloptions ) {
 		$suppress      = $wpdb->suppress_errors();
-		$alloptions_db = $wpdb->get_results( "SELECT option_name, option_value FROM $wpdb->options WHERE autoload = 'yes'" );
+		$alloptions_db = $wpdb->get_results( "SELECT option_name, option_value FROM $wpdb->options WHERE autoload = 'yes' OR autoload = 'default-yes'" );
```

Comment:
Backwards compatiblity might be an issue here.

https://www.wpdirectory.net/search/01HASJYW8KEYKH2AKFP00SNMXF